ssv/runner_validations.go

```diff
@@ -50,7 +50,7 @@ func (b *BaseRunner) ValidatePostConsensusMsg(runner Runner, psigMsgs *types.Par
 	}
 
 	// TODO https://github.com/ssvlabs/ssv-spec/issues/142 need to fix with this issue solution instead.
-	if b.State.DecidedValue == nil || len(b.State.DecidedValue) == 0 {
```

Comment:
that's from another PR no?
you can resolve when solving conflicts

Reply:
Due to the linter update, I have been fixing this lint issue in all the recent PRs